lib/styles/components/_items-list-item.scss

```diff
@@ -26,9 +26,15 @@
 		width: 100%;
 	}
 
+	&__label {
+		flex: 1 1 auto;
+		width: 50%;
+	}
+
 	&__value {
 		color: var(--raw-gray-600);
 		flex: 1 1 auto;
+		width: 50%;
```

Comment:
it's funny that we use `50%` for 3-column layout. Did you consider using table for these pages?

Reply:
I convert it to the table. I know it looks funny, but it works fine: if we have 2 columns or 3 they are always equal.